action.yaml

```diff
@@ -10,46 +10,48 @@ branding:
   icon: "package" # icon name from Feather open source icons pack
   color: "orange"
 
-permissions:
-  contents: read
-  
-inputs: 
-  REPO_TOKEN:
-    required: true
+inputs:
   DEPENDENCY_FILES:
+    description: A string params passed to Packj for auditing
     required: true
-    
+  REPO_TOKEN:
+    description: Your repo GITHUB_TOKEN
+    required: true
+
 runs:
-  using: composite
-  steps: 
+  using: "composite"
 
+  steps:
     # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
-    - name: Checkout repo
+    - name: Check out repository code
       uses: actions/checkout@v3
 
-    # Audit deps
+    - name: Setup Python & pip
+      uses: actions/setup-python@v4
+      with:
+        python-version: "3.10"
+        cache: pip
+
+    - name: Clone Packj repo
+      shell: bash
+      run: git clone https://github.com/ossillate-inc/packj
+
+    - name: Create virtual environment
+      shell: bash
+      run: python3 -m venv venv
+
+    - name: Activate virtual environment
+      shell: bash
+      run: source venv/bin/activate
+
+    - name: Install Packj's dependencies
+      shell: bash
+      run: pip3 install -r ./packj/requirements.txt
+
     - name: Auditing deps with Packj
       shell: bash
-      run: |
```

Comment:
We need correct path to dependencies. That's exactly what this piece of code does. Please add it back.